pkg/oci/oci.go

```diff
@@ -610,3 +610,97 @@ func getImageManifest(ctx context.Context, target oras.Target, image string, aut
 	}
 	return manifestHost, nil
 }
+
+// EnsureImage ensures the image is present in the local store
+func EnsureImage(ctx context.Context, image string, authOpts *AuthOptions, pullPolicy string) error {
```

Comment:
`GetGadgetImage` (line 94) and `EnsureImage` have similar code. Can this be factored?